contracts/examples/chainlink-solana-demo/programs/chainlink_solana_demo/src/lib.rs

```diff
@@ -50,14 +52,26 @@ pub mod hello_world {
             ctx.accounts.chainlink_feed.to_account_info(),
         )?;
 
-        let decimal = Decimal::new(round.answer, u32::from(decimals));
-        msg!("{} price is {}", description, decimal);
+        // Set the account value
+        let decimal: &mut Account<Decimal> = &mut ctx.accounts.decimal;
+        decimal.value=round.answer;
+        decimal.decimals=u32::from(decimals);
+
```

Comment:
You will need to `cargo fmt`, the code isn't properly formatted. Tests didn't seem to run.